forge/test/mlir/test_ops.py

```diff
@@ -633,3 +633,54 @@ def forward(self, input_tensor, indices):
     # Validate results
     assert compare_with_golden_pcc(golden=golden_out.value(), calculated=co_out[0], pcc=0.99)
 
+
+@pytest.mark.parametrize("input_shape", [
+    (2, 32, 64, 64), 
+    (3, 22, 37, 41),
+    (2, 32, 64),
+    (3, 22, 37),
+], ids=["(2, 32, 64, 64)", "(2, 22, 37, 41)", "(2, 32, 64)", "(3, 22, 37)"])
```

Comment:
Let's push tests without these IDs for now. Our testing FW should have a more suitable approach for handling them :))

Reply:
Okay. I will remove those ids from test.

Reply:
@nvukobratTT Removed the ids from the pytest